pennylane/measurements/mid_measure.py

```diff
@@ -28,8 +28,11 @@ def measure(wires: Wires, reset: Optional[bool] = False, postselect: Optional[in
     r"""Perform a mid-circuit measurement in the computational basis on the
     supplied qubit.
 
+    Computational basis measurements are performed using the 0, 1 convention
+    rather than the ±1 convetion. 
```

Comment:
```suggestion
    Computational basis measurements are performed using the 0, 1 convention
    rather than the ±1 convention. 
```

Reply:
I'm fine with this addition, although I'm wondering whether the relationship to the original issue (`qml.measure` <> `qml.PauliZ()`) is clear to users a priori? 
We could also add a comment in the details section that makes the confusion explicit and shows that the difference between the two is by design, not a bug :thinking:

Reply:
I'm not sure that the details section (under postselection) makes sense for an explicit example. Maybe above it?